distribution/macos/create_macos_release.sh

```diff
@@ -7,54 +7,54 @@ if [ "$#" -ne 6 ]; then
     exit 1
 fi
 
-mkdir -p $1
-mkdir -p $2
-mkdir -p $3
-
-BASE_DIR=$(readlink -f $1)
-TEMP_DIRECTORY=$(readlink -f $2)
-OUTPUT_DIRECTORY=$(readlink -f $3)
-ENTITLEMENTS_FILE_PATH=$(readlink -f $4)
+mkdir -p "$1"
+mkdir -p "$2"
+mkdir -p "$3"
+
+BASE_DIR=$(readlink -f "$1")
+TEMP_DIRECTORY=$(readlink -f "$2")
+OUTPUT_DIRECTORY=$(readlink -f "$3")
+ENTITLEMENTS_FILE_PATH=$(readlink -f "$4")
 VERSION=$5
 SOURCE_REVISION_ID=$6
 
 RELEASE_TAR_FILE_NAME=Ryujinx-$VERSION-macos_universal.app.tar
-ARM64_APP_BUNDLE=$TEMP_DIRECTORY/output_arm64/Ryujinx.app
-X64_APP_BUNDLE=$TEMP_DIRECTORY/output_x64/Ryujinx.app
-UNIVERSAL_APP_BUNDLE=$OUTPUT_DIRECTORY/Ryujinx.app
+ARM64_APP_BUNDLE="$TEMP_DIRECTORY"/output_arm64/Ryujinx.app
+X64_APP_BUNDLE="$TEMP_DIRECTORY"/output_x64/Ryujinx.app
+UNIVERSAL_APP_BUNDLE="$OUTPUT_DIRECTORY"/Ryujinx.app
 EXECUTABLE_SUB_PATH=Contents/MacOS/Ryujinx
 
-rm -rf $TEMP_DIRECTORY
-mkdir -p $TEMP_DIRECTORY
+rm -rf "$TEMP_DIRECTORY"
+mkdir -p "$TEMP_DIRECTORY"
 
 DOTNET_COMMON_ARGS="-p:DebugType=embedded -p:Version=$VERSION -p:SourceRevisionId=$SOURCE_REVISION_ID --self-contained true"
 
 dotnet restore
 dotnet build -c Release Ryujinx.Ava
-dotnet publish -c Release -r osx-arm64 -o $TEMP_DIRECTORY/publish_arm64 $DOTNET_COMMON_ARGS Ryujinx.Ava
-dotnet publish -c Release -r osx-x64 -o $TEMP_DIRECTORY/publish_x64 $DOTNET_COMMON_ARGS Ryujinx.Ava
+dotnet publish -c Release -r osx-arm64 -o "$TEMP_DIRECTORY"/publish_arm64 $DOTNET_COMMON_ARGS Ryujinx.Ava
+dotnet publish -c Release -r osx-x64 -o "$TEMP_DIRECTORY"/publish_x64 $DOTNET_COMMON_ARGS Ryujinx.Ava
 
 # Get ride of the support library for ARMeilleur for x64 (that's only for arm64)
-rm -rf $TEMP_DIRECTORY/publish_x64/libarmeilleure-jitsupport.dylib
+rm -rf "$TEMP_DIRECTORY"/publish_x64/libarmeilleure-jitsupport.dylib
```

Comment:
```suggestion
rm -rf "$TEMP_DIRECTORY/publish_x64/libarmeilleure-jitsupport.dylib"
```
You could also fix the typo above if you want. "Get rid" instead of "Get ride".